ofrak_core/src/ofrak/resource.py

```diff
@@ -1253,6 +1279,9 @@ async def get_only_descendant_as_view(
         ``r_filter``
         :raises NotFoundError: If a filter is not provided and this resource has multiple descendant
         """
+        await self.get_descendants_as_view(
```

Comment:
Should the docstring be updated to say that the views are first created? As a way to distinguish this with `get_descendants`....

Reply:
This comment applies everywhere. What changes, if any, should be made in the docstrings to make it clear that:

1. `*as_view*` will create views first, then filter
2. `get_*` will just apply filter without doing analysis first.